cl/antiquary/state_antiquary.go

```diff
@@ -360,9 +360,11 @@ func (s *Antiquary) IncrementBeaconState(ctx context.Context, to uint64) error {
 	defer progressTimer.Stop()
 	prevSlot := slot
 	first := false
+	blocksBeforeCommit := 100_000
+	blocksProcessed := 0
 	// This tells us that transition and operations do not happen concurrently and access is safe, so we can optimize for GC.
 	// there is optimized custom cache to recycle big GC overhead.
-	for ; slot < to; slot++ {
+	for ; slot < to && blocksProcessed < blocksBeforeCommit; slot++ {
```

Comment:
I have a question. Why did you decide to limit `blocksProcessed` exactly to 100k?

Reply:
just arbitrary, so that the save on disk every 100k blocks. that would be 100k(blk)/33(blk/sec) = 56 minutes